programs/ziti-edge-tunnel/ziti-edge-tunnel.c

```diff
@@ -48,6 +48,10 @@
 #define MAXMESSAGELEN 4096
 #endif
 
+#ifndef MAXIPCCOMMANDLEN
+#define MAXIPCCOMMANDLEN 4096 * 4
```

Comment:
can't we just put the one if win32 here? are all these other changes actually necessary if we just make the buffer bigger for win32?

Reply:
I kinda expected this to be just a single change somewhere - why all the 'extra code'? Also we really need to get away from littering the #if _WIN32 everywhere - there's a lot of them in here ...

Reply:
I moved the MAXIPCCOMMANDLEN macro into the win32 block. I could not find an in built option to increase the buffer length in the uv_read function.

Reply:
I also don't see why this shouldn't be a single-line (buffer size) change for all operating systems.

Reply:
I tested the long jwt value in linux, the ipc command didnt get split after 1024 bytes in linux. Probably this buffering issue after 1024 bytes, might be occurring only in windows names pipe, ZDE is not throwing that error in linux when I tested it.